resources/uk/gov/hmcts/helm/push-helm-charts-to-git.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -x
+
+CHART_DIRECTORY=${1}
+CHART_NAME=${2}
+USER_NAME=${3}
+EMAIL_ID=${4}
+VERSION=${5}
+
+if [ -d "hmcts-charts" ]; then
+  rm -rf hmcts-charts
+fi
+
+git clone git@github.com:hmcts/hmcts-charts.git
```

Comment:
I don't think this will work on jenkins, you will want to clone over https afaik, https://github.com/hmcts/cnp-jenkins-library/blob/master/resources/uk/gov/hmcts/helm/check-helm-version-bumped.sh#L58